phpcs.xml.dist

```diff
@@ -1,108 +1,30 @@
 <?xml version="1.0"?>
-<ruleset name="WordPress Coding Standards for Performance Plugin">
-	<description>Sniffs for WordPress plugins, with minor modifications for Performance</description>
-
-	<rule ref="PHPCompatibility"/>
-	<config name="testVersion" value="7.0-"/>
-
-	<rule ref="WordPress-Docs"/>
-	<rule ref="WordPress-Extra" />
-	<rule ref="WordPress.WP.I18n"/>
-	<config name="text_domain" value="performance-lab,default,auto-sizes"/>
-
-	<arg value="ps"/>
-	<arg name="extensions" value="php"/>
-
-	<file>./admin</file>
-	<file>./load.php</file>
-	<file>./modules</file>
-	<file>./plugins</file>
-	<file>./server-timing</file>
-	<file>./tests</file>
+<ruleset name="WPP-PerfLab">
+	<description>WordPress Coding Standards for the Performance Lab Plugin</description>
+	<rule ref="phpcs.ruleset.xml"/>
+	
+	<config name="text_domain" value="performance-lab"/>
```

Comment:
Why not keep `default` in here as it has been so far? That's not related to a specific plugin, but intended for strings that are found exactly like that in WordPress core. Using that approach is totally reasonable within a plugin, so I think we should keep that text domain included here rather than working around it with exceptions.
```suggestion
	<config name="text_domain" value="performance-lab,default"/>
```

Reply:
Added "default" text domain back in https://github.com/WordPress/performance/pull/1002/commits/4927e3c518cdc03ec91295d11632826361e22545